packages/react-components/react-infolabel/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@fluentui/react-infolabel-preview",
-  "version": "0.1.0",
+  "name": "@fluentui/react-infolabel",
+  "version": "9.0.0-alpha.0",
```

Comment:
is -alpha.0 the right version number here?

Reply:
When release is triggered, beachball will bump this from `9.0.0-alpha.0` to `9.0.0`

Reply:
the change file will make a minor bump, this will update the version to 9.0.0